winui/Kanban/Getting-started.md

```diff
@@ -0,0 +1,241 @@
+            taskDetail.Image = new Image
+            {
+                Source = new BitmapImage(new Uri(path + "Assets/Kanban/People_Circle4.png"))
+            };
+
+            taskDetails.Add(taskDetail);
+            return taskDetails;
+        }
+
+        #endregion
+    }
+
+{% endhighlight %}
+
+### Bind Data to the Kanban Control
+
+To bind a data source to the `SfKanban` control, set the `ItemsSource` property as demonstrated in the example below. This property connects the collection of `KanbanModel` objects to the Kanban board, enabling the display of tasks.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<Grid>
+    <kanban:SfKanban x:Name="sfKanban"
+                     ItemsSource="{Binding TaskDetails}"/>
+    <Grid.DataContext>
+        <local:GettingStartedViewModel/>
+    </Grid.DataContext>
+</Grid>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+this.sfKanban.ItemsSource = new GettingStartedViewModel().TaskDetails;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Define columns
+
+To configure columns in the `SfKanban` control, you must manually define them by adding `KanbanColumn` objects to the `Columns` collection property of SfKanban.
+
+The `ItemsSource` bound to the Kanban control will populate the respective columns based on the `Categories` property in KanbanColumn.
+
+The following code example illustrates how this can be done.
+
+{% tabs %}
+{% highlight xaml %}
+
+<kanban:SfKanban x:Name="sfKanban"
+                 ItemsSource="{Binding TaskDetails}"
+                 AutoGenerateColumns="False">
+    <kanban:KanbanColumn Categories="Open" HeaderText="To Do"/>
+    <kanban:KanbanColumn Categories="In Progress" HeaderText="Progress"/>
+    <kanban:KanbanColumn Categories="Review,Closed" HeaderText="Done"/>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+this.sfKanban.ItemsSource = new GettingStartedViewModel().TaskDetails;
+this.sfKanban.AutoGenerateColumns = false;
+
+sfKanban.Columns.Add(new KanbanColumn() { Categories = "Open", HeaderText = "To Do", });
+sfKanban.Columns.Add(new KanbanColumn() { Categories = "In Progress", HeaderText = "Progress", });
+sfKanban.Columns.Add(new KanbanColumn() { Categories = "Review,Closed", HeaderText = "Done", });
+
+{% endhighlight %}
+{% endtabs %}
+
+Alternatively, you can set the `AutoGenerateColumns` property to `true` to automatically create columns without manually defining them. When enabled, the columns are generated dynamically based on the `Category` property of KanbanModel, creating a column for each distinct value in the `ItemsSource`.
+
+## Theme
+
+The WinUI Kanban supports light and dark themes, automatically adjusting to the system's theme settings for a consistent and visually appealing experience in any environment.
+
+Refer to the following link for guidance on applying themes: [Themes for Syncfusion WinUI controls](https://help.syncfusion.com/winui/common/themes)
+
+You can find the kanban keys for both themes here.
```

Comment:
Add below 

**Note**

You can refer to our `WinUI Kanban` feature tour page for its groundbreaking feature representations. You can also explore our `WinUI Kanban Example` that shows you how to render the Kanban in WinUI.

Reply:
Content changed.